beacon-chain/utils/shuffle.go

```diff
@@ -2,57 +2,52 @@
 package utils
 
 import (
-	"errors"
+	"encoding/binary"
+	"math"
 
 	"github.com/ethereum/go-ethereum/common"
 	"github.com/prysmaticlabs/prysm/shared/hashutil"
-	"github.com/prysmaticlabs/prysm/shared/params"
+	//"github.com/prysmaticlabs/prysm/shared/params"
 )
 
 // ShuffleIndices returns a list of pseudorandomly sampled
 // indices. This is used to shuffle validators on ETH2.0 beacon chain.
 func ShuffleIndices(seed common.Hash, indicesList []uint64) ([]uint64, error) {
```

Comment:
This is confusing this doesnt match the spec at all , this is the function in the spec
```py
def get_permuted_index(index: int, list_size: int, seed: Bytes32) -> int:
```
There also no assertions from the spec 
```py
    assert index < list_size
    assert list_size <= 2**40
```